docs/EmbeddedAppIOS.md

```diff
@@ -11,11 +11,26 @@ Since React makes no assumptions about the rest of your technology stack – it
 
 ## Requirements
 
+- Xcode project - If you don't have native app, create a sample Xcode project now. 
 - [CocoaPods](http://cocoapods.org/) – `gem install cocoapods`
 - [Node.js](http://nodejs.org)
   - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
+- Create a `package.json` file in your root directory with these contents:
+```
+{
+  "name": "embedded",
+  "version": "1.0.0",
+  "description": "Integrating React Native with Existing Apps",
+  "main": "index.js",
+  "homepage": "https://facebook.github.io/react-native/docs/embedded-app-ios.html",
+  "dependencies": {
+    "react-native": "^0.26.0",
+    "react": "~15.0.2"
+  }
+}
+```
 - Install the `react-native` package from npm by running the following command in the root directory of your project:
-  - `npm install react-native`
```

Comment:
Hey I think that it ultimately depends on when these docs will be released. For now, following these docs on version 0.26.x will throw a warning like this:
`react-native@0.26.3 requires a peer of react@15.0.2 but none was installed.`
And if the user will not run `npm i react@15.0.2 --save --save-exact` he will definitely run into problems.
@javache We might want to add a reference to this PR on the release notes. 
And for the future as @davidshayer said, we do need to add the following command sequence:
```
npm init
npm install react --save
npm install react-native --save
```
Great day,
David